tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -158,8 +155,7 @@ Result:
 
 Arc polygon function
 ^^^^^^^^^^^^^^^^^^^^
-
-Only for example purpose, we also provide here another function, very similar, that draws the plain portion of the disc defined by the arc, not only its shape. The method is exactly the same a previously, except that we draw a polygon instead of lines.
+We can take this a step further and write a function that draws the plain portion of the disc defined by the arc, not only its shape. The method is exactly the same a previously, except that we draw a polygon instead of lines:
```

Comment:
Please make sure to keep one empty newline after a header, reST is touchy. I think it works without here, but it's better to stay consistent.